src/coreComponents/constitutive/solid/PorousSolid.hpp

```diff
@@ -87,6 +87,90 @@ class PorousSolidUpdates : public CoupledSolidUpdates< SOLID_TYPE, BiotPorosity,
                                                   dPorosity_dPressure,
                                                   dPorosity_dVolStrain );
 
+// TODO uncomment once we start using permeability model in flow.
+//    m_permUpdate.updateFromPressureStrain( k,
+//                                           q,
+//                                           pressure,
+//                                           volStrain );
+  }
+
+  GEOSX_HOST_DEVICE
+  void smallStrainUpdateSinglePhase( localIndex const k,
+                                     localIndex const q,
+                                     real64 const & fluidPressure,
```

Comment:
We should enforce a consistent naming convention, like `fluidPressureOld`, to clarify at what time step some of these quantities exist.  c.f. `fluidDensityOld`.